modules/color/color.graphql.js

```diff
@@ -4,6 +4,7 @@ const colorType = `
     name: [Language]
     colorHex: String
     simpleName: [Language]
+    translations_key: ID
```

Comment:
try to use camelCase instead of snake_case. wait for hot fix maybe....

Reply:
Sorry, but for now we should use this approach and after everyone will complete similar tasks, we are gonna add "translations" camelCase fix